lib/Doctrine/DBAL/Migrations/SqlFileWriter.php

```diff
@@ -107,7 +107,6 @@ private function buildMigrationFilePath()
     {
         $path = $this->destPath;
         if (is_dir($path)) {
-            $path = realpath($path);
```

Comment:
I need to remove the realpath call because https://github.com/mikey179/vfsStream/wiki/Known-Issues

I hope that adding this dependency is not a problem.